packages/config/src/Manufacturers.ts

```diff
@@ -93,3 +94,47 @@ export function lookupManufacturer(manufacturerId: number): string | undefined {
 
 	return manufacturers.get(manufacturerId);
 }
+
+/**
+ * Add new manufacturers to configuration DB
+ * @param manufacturerId The manufacturer id to look up
+ * @param manufacturerName The manufacturer name
+ */
+export function setManufacturer(
+	manufacturerId: number,
+	manufacturerName: string,
+): void {
+	if (!manufacturers) {
+		throw new ZWaveError(
+			"The config has not been loaded yet!",
+			ZWaveErrorCodes.Driver_NotReady,
+		);
+	}
+
+	manufacturers.set(manufacturerId, manufacturerName);
+}
+
+/**
+ * Write current manufacturers map to json
+ */
+export async function writeManufacturersToJson(): Promise<void> {
+	if (!manufacturers) {
+		throw new ZWaveError(
+			"The config has not been loaded yet!",
+			ZWaveErrorCodes.Driver_NotReady,
+		);
+	}
+
+	const data: Record<string, string> = {};
+
+	const orderedMap = new Map(
+		[...manufacturers].sort((a, b) => (a[0] > b[0] ? 1 : -1)),
```

Comment:
I think you're supposed to return 0 if both are equal

Reply:
In theory yes, but there couldn't be 2 manufacturers with the same id